webapp/app.yaml

```
@@ -38,18 +35,7 @@ handlers:
 
 # Don't upload mock-data static files to AppEngine
 skip_files:
```

```suggestion
skip_files:
- bower_components/
```

If the only thing using bower now is `wct`, we can and should skip the whole `bower_components` directory when deploying.